### PR TITLE
Change name zenchef

### PR DESCRIPTION
**Pull request by adeclosset**
@samjewell - Hey Sam, quick favour. Had to change the name of 1001 Menus to Zenchef following their series B & name change. Can you quickly check if all is good?

### CODE REVIEW

File: app/pages/stories/stories/zenchef.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ export default class Stories1001Menus extends React.Component {
         ]}>
         <StoryQuote tag='Le secteur des paiements est dominé par des logiciels complexes et archaïque'>
           <p className='para u-margin-Tn'>
-            <a href='http://www.1001menus.com' className='u-link-color-p u-text-underline' target='_blank'>
-            1001 Menus</a>, un outil tout-en-un pour gérer la communication de restaurants,
+            <a href='http://www.zenchef.com' className='u-link-color-p u-text-underline' target='_blank'>
+            Zenchef</a>, un outil tout-en-un pour gérer la communication de restaurants (ex-1001 Menus),
             utilise le prélèvement pour encaisser les abonnements mensuels de ses clients, permettant
             aux restaurateurs de ne pas avoir à se soucier d'envoyer à temps un chèque ou un virement.
```
Comment (samjewell) [inline app/pages/stories/stories/zenchef.js]: Can we have a directional apostrophe here? `d’envoyer`

**Comment by samjewell** [comment]
Hey man. The diff looks good to me. Have you checked to make sure that `1001` and `menus` have been completely removed from the repo now? (other than your ex-1001menus comment).

You have merge conflicts which need resolving, and also you should probably rebase against the latest `dev` in order to get rid of that crazy commit history...

**Comment by samjewell** [comment]
Once you've rebased against dev, resolved merge conflicts, and added that apostrophe, I'm  :+1:

**Comment by adeclosset** [comment]
@samjewell - all done, thanks to @stevedomin heroic help!

**Pull request merged**